src/phongo_compat.h

```diff
@@ -285,15 +285,6 @@ static inline zend_bool zend_ini_parse_bool(zend_string* str)
 	Z_PARAM_ARRAY_OR_OBJECT(dest)
 #endif
 
-/* Per https://wiki.php.net/rfc/internal_method_return_types, "Non-final
- * internal method return types - when possible - are declared tentatively in
- * PHP 8.1, and they will become enforced in PHP 9.0." This can be revisited
- * when more general typing improvements are made in PHPC-1709. */
```

Comment:
[PHPC-1709](https://jira.mongodb.org/browse/PHPC-1709) has long since been resolved. Glad to see this removed :)